### ISSUE TITLE
[Accessibility] Month Navigation is Unnavigable with iOS VoiceOver activated

### ISSUE THREAD

**Issue by danielschnoll** [wix/react-native-calendars#2784]
## Issue

We use React Native Calendars on our mobile app, and are in the midst of an Accessibility improvement effort. We found that when VoiceOver is active on an iOS device, the caret icons `<` and `>` for navigating to a new month are unreachable by VoiceOver, and swiping gestures are unable to change the months as well.

This issue is not present on Android devices. TalkBack is able to focus on the buttons and toggle the months normally.

## Evidence

https://github.com/user-attachments/assets/debdeae2-8aff-4ea1-b2f1-451ffc22c563

## Fix

We created a React Native patch that appears to fix the issue and allow for a more logical focus order on the Calendar month selection header

**Comment by danielschnoll** [comment]
Turns out this is a non-issue. The month changing is accessible by swiping up/down, which we didn't realize until just a short bit ago. Closing this issue.
